libraries/dagster-teradata/README.md

```diff
@@ -0,0 +1,138 @@
+# dagster-teradata
+
+A dagster module that provides integration with [Teradata Vantage](https://www.teradata.com/).
+
+## Installation
+The `dagster_teradata` module is available as a PyPI package - install with your preferred python 
+environment manager.
+
+```
+source .venv/bin/activate
+pip install dagster-teradata
+```
+
+## Example Usage
+
+This offers seamless integration with Teradata Vantage, facilitating efficient workflows for data processing, management, 
+and transformation. This module supports a range of scenarios, such as executing queries, managing tables, 
+and integrating with cloud storage solutions like AWS S3 and Azure Data Lake Storage (ADLS). Additionally, 
+it enables compute cluster management for Teradata Vantage Cloud Lake.
+
+```python
+import os
+import pytest
+from dagster import job, op
+from dagster_teradata import TeradataResource
+
+td_resource = TeradataResource(
+    host=os.getenv("TERADATA_HOST"),
+    user=os.getenv("TERADATA_USER"),
+    password=os.getenv("TERADATA_PASSWORD"),
+    database=os.getenv("TERADATA_DATABASE"),
+)
+
+@pytest.mark.integration
+def test_execute_query(tmp_path):
+    @op(required_resource_keys={"teradata"})
+    def example_test_execute_query(context):
+        result = context.resources.teradata.execute_queries(
+            ["select order_id from orders_24", "select order_id from orders_25"], True
+        )
+        context.log.info(result)
+
+    @job(resource_defs={"teradata": td_resource})
+    def example_job():
+        example_test_execute_query()
+
+    example_job.execute_in_process(resources={"teradata": td_resource})
```

Comment:
For examples, please remove `pytest` definitions, and show only the `op`, `'job`, or `asset`.